test/e2e/nodenetworkstate_conditions_test.go

```diff
@@ -0,0 +1,92 @@
+package e2e
+
+import (
+	"time"
+
+	. "github.com/onsi/ginkgo"
+
+	corev1 "k8s.io/api/core/v1"
+
+	nmstatev1alpha1 "github.com/nmstate/kubernetes-nmstate/pkg/apis/nmstate/v1alpha1"
+)
+
+var _ = Describe("NodeNetworkStateCondition", func() {
+	var (
+		br1Up = nmstatev1alpha1.State(`interfaces:
+  - name: eth1
+    type: ethernet
+    state: up
+  - name: br1
+    type: linux-bridge
+    state: up
+    bridge:
+      options:
+        stp:
+          enabled: false
+      port:
+        - name: eth1
+          stp-hairpin-mode: false
```

Comment:
There was an issue where we had to set stp even if disabled. Could you try to drop all stp options from here and check if the test still passes?